third_party/move/tools/move-unit-test/src/lib.rs

```diff
@@ -187,7 +189,7 @@ impl UnitTestingConfig {
         let (units, warnings) =
             diagnostics::unwrap_or_report_diagnostics(&files, compilation_result);
         diagnostics::report_warnings(&files, warnings);
-        test_plan.map(|tests| TestPlan::new(tests, files, units))
+        test_plan.map(|tests| TestPlan::new(tests, files, units, Default::default()))
```

Comment:
Nit: might be clearer to explicitly use empty vector instead of `Default::default()` here.